daemons/based/pacemaker-based.c

```diff
@@ -383,13 +383,13 @@ cib_init(void)
 {
     crm_cluster = pcmk_cluster_new();
 
-    if (is_corosync_cluster()) {
 #if SUPPORT_COROSYNC
+    if (is_corosync_cluster()) {
         crm_cluster->destroy = cib_cs_destroy;
         crm_cluster->cpg.cpg_deliver_fn = cib_cs_dispatch;
         crm_cluster->cpg.cpg_confchg_fn = pcmk_cpg_membership;
-#endif
     }
+#endif // SUPPORT_COROSYNC
```

Comment:
FYI, is_corosync_cluster() is intentionally designed to be usable without guards, but this case makes sense since we're guarding the contents.

Reply:
I noticed that. This commit was only about consistency. We guard `is_corosync_cluster()` in a lot of places.